src/main/java/com/mongodb/jdbc/MongoVersionedJsonSchema.java

```diff
@@ -21,7 +21,6 @@ public MongoVersionedJsonSchema(
             @BsonProperty("version") final int version,
             @BsonProperty("jsonSchema") JsonSchema schema) {
         this.version = version;
-        System.out.println(schema);
```

Comment:
I found another debug statement we missed... I think.

@nbagnard tell me if this was actually intended to be here!